README.md

```diff
@@ -1,43 +1,49 @@
+<a href="https://www.twilio.com">
+  <img src="https://static0.twilio.com/marketing/bundles/marketing/img/logos/wordmark-red.svg" alt="Twilio" width="250" />
+</a>
+
 # Verify Quickstart
 
+![](https://github.com/TwilioDevEd/verify-v2-quickstart-python/workflows/Flask/badge.svg)
+
 > We are currently in the process of updating this sample template. If you are encountering any issues with the sample, please open an issue at [github.com/twilio-labs/code-exchange/issues](https://github.com/twilio-labs/code-exchange/issues) and we'll try to help you.
 
-Simple phone verification with Python, Flask, and Twilio Verify. 
+Simple phone verification with Python, Flask, and Twilio Verify.
 
-Full Quickstart instructions available at https://www.twilio.com/docs/verify/api-beta/quickstarts/python-flask
+[Read the full quickstart here](https://www.twilio.com/docs/verify/api-beta/quickstarts/python-flask)!
 
 ## Install
 
+    ```
     python3 -m venv venv
     . venv/bin/activate
+    ```
 
 Or on Windows cmd:
 
+    ```
     py -3 -m venv venv
     venv\Scripts\activate.bat
+    ```
 
-Install Verify:
+Install the dependencies:
 
+    ```
     pip install -r requirements.txt
+    ```
 
 ## Run
 
-    export FLASK_APP=verify
-    export FLASK_ENV=development
```

Comment:
are these no longer necessary? the app doesnt seem to work for me without them

Reply:
They are located in the `.env` file and autoloader. I'll add the instructions to copy the .env.example and replace the values.